src/mon/OSDMonitor.cc

```diff
@@ -4384,6 +4384,22 @@ void OSDMonitor::get_pools_health(
 	detail->push_back(make_pair(HEALTH_WARN, ss.str()));
     }
 
+    if (1 == pool.size) {
+      stringstream ss;
+      ss << "pool '" << pool_name << "' size is 1 which not safe enough ";
+      summary.push_back(make_pair(HEALTH_WARN, ss.str()));
+      if (detail)
+        detail->push_back(make_pair(HEALTH_WARN, ss.str()));
+    }
+
+    if (1 == pool.min_size) {
+      stringstream ss;
+      ss << "pool '" << pool_name << "' min_size is 1 which not safe enough ";
+      summary.push_back(make_pair(HEALTH_WARN, ss.str()));
+      if (detail)
+        detail->push_back(make_pair(HEALTH_WARN, ss.str()));
```

Comment:
Use "summary" for short description (i.e. pool "abc" min_size is 1), and "detail" for more verbose info ("size 1 doesn't offer any data safety").